distribution/deb/src/main/packaging/init.d/elasticsearch

```diff
@@ -109,8 +109,10 @@ export ES_GC_LOG_FILE
 export JAVA_HOME
 export ES_INCLUDE
 
-# Check DAEMON exists
-test -x $DAEMON || exit 0
+if [ ! -x "$DAEMON" ] {
+	echo "Could not find elasticsearch executable binary."
```

Comment:
The test is whether or not the elasticsearch *script* (not a *binary*; standard Java applications do not have executable binaries anyway) exists and is executable. Can you change this to say something along the lines of "the elasticsearch script does not exist or is not executable"?